src/generator/README.md

```diff
@@ -0,0 +1,13 @@
+# Unicode specification extraction
+
+The file `src/syntax/unicode.ml` is generated using the data available at [unicode.org](https://www.unicode.org/Public/).
+
+The rule at `src/syntax/dune` is the main entry point for this process. It triggers:
+* download of data files at `src/generator/data`
+* build of `src/generator/gen_unicode.exe`
+* generation `unicode.ml` and places a copy in the source tree and a copy in the build tree
+
+The rule is ignoed when using the `--ignore-promoted-rules` option. This option is also implied when using
+`-p`/`--for-release-of-packages`.
```

Comment:
So if I'm a maintainer and I want to update the Unicode version, what do I do? You might want to specify an explicit list of steps (as in "run this to generate a new unicode.ml, then run this to test that it's okay, then check just the new unicode.ml into git...")

Reply:
BTW, I may have the steps completely wrong there. (Which is part of why documentation is good!)